code/__DEFINES/traits.dm

```diff
@@ -291,6 +290,33 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_KLEPTOMANIAC 		"kleptomaniac"
 //MonkeStation Edit End
 
+/*
+* Trait granted by various security jobs, and checked by [/obj/item/food/donut]
+* When present in the mob's mind, they will always love donuts.
+*/
+#define TRAIT_DONUT_LOVER "donut_lover"
+
+// METABOLISMS
+// Various jobs on the station have historically had better reactions
+// to various drinks and foodstuffs. Security liking donuts is a classic
+// example. Through years of training/abuse, their livers have taken
+// a liking to those substances. Steal a sec officer's liver, eat donuts good.
+
+// These traits are applied to /obj/item/organ/liver
+#define TRAIT_LAW_ENFORCEMENT_METABOLISM "law_enforcement_metabolism"
+#define TRAIT_CULINARY_METABOLISM "culinary_metabolism"
+#define TRAIT_COMEDY_METABOLISM "comedy_metabolism"
+#define TRAIT_MEDICAL_METABOLISM "medical_metabolism"
+#define TRAIT_GREYTIDE_METABOLISM "greytide_metabolism"
+#define TRAIT_ENGINEER_METABOLISM "engineer_metabolism"
+#define TRAIT_ROYAL_METABOLISM "royal_metabolism"
+#define TRAIT_PRETENDER_ROYAL_METABOLISM "pretender_royal_metabolism"
```

Comment:
Do all of these have enough uses to make them worth being dedicated livers/traits?

Reply:
TRAIT_PRETENDER_ROYAL_METABOLISM  is used for qm as a jab at him being a fake head TRAIT_ROYAL_METABOLISM is for actual heads TRAIT_ENGINEER_METABOLISM  is for engineers TRAIT_GREYTIDE_METABOLISM  is for assistants, making them traits is by far the easier way of doing this as it means that if we add proper skill chip systems for other organs for job changes we can add the traits to people's livers.

Reply:
the royal traits currently do nothing aside from adding lore to the livers, but i plan on adding more drinks in the future that affect them its just out of this pr's scope